### PR TITLE
TUTORIAL: Fix #3965 Corrected tutorial text

### LINKED ISSUE #3965: I met the wrong tutorial guide and found something can make user confused (v2.0)
I played it for the first time today.
I found something strange while doing a tutorial on hacking the n00dles server.

The n00dles server has 4GB of RAM, but the tutorial guide said 8GB.
I checked which one was wrong, the RAM of the n00dle server or the guide of the tutorial.

image

This is circumstantial that there is a problem with the tutorial guidance.
Cuz I found when I used `free`, `analyze`, and `scan-analyze 2` all showed 4.00GB of RAM.

image

image

---

I also don't know why tutorial guide sometimes display terminal username `home` instead of `n00dles` when it still proceeding with tutorials working on `n00dles`'s terminal. I thought I had to go `home` again.
I'm a beginner, so I was little confused by this.

image

image

### PR DESCRIPTION
**Pull request by mihilt**
close #3965

### CODE REVIEW

**Comment by Snarling** [comment]
Actual changes look good to me. Title name should be reformatted to be more informative and follow the template. Example: "TUTORIAL: Fix #3965 Corrected tutorial text"

File: src/ui/InteractiveTutorial/InteractiveTutorialRoot.tsx [hunk 1]
```diff
@@ -376,12 +376,12 @@ export function InteractiveTutorialRoot(): React.ReactElement {
             <br />
             To create a new script or edit an existing one, you can use{" "}
           </Typography>
-          <Typography classes={{ root: classes.textfield }}>{"[home ~/]> nano"}</Typography>
+          <Typography classes={{ root: classes.textfield }}>{"[n00dles ~/]> nano"}</Typography>
```
Comment (phyzical) [inline src/ui/InteractiveTutorial/InteractiveTutorialRoot.tsx:379]: wouldnt  these be on home not n00dles though as the previous step says `Let's head home before creating our first script!`

Comment (mihilt) [reply src/ui/InteractiveTutorial/InteractiveTutorialRoot.tsx]: Yup thank you for letting me know and reviewing it!

**Comment by phyzical** [comment]
@mihilt sorry i think this correction would apply to all the noodles changes not just the one i linked

**Pull request merged**